lib/datadog/tracing/contrib/qless/integration.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
```

Comment:
Is this file and the qless_job.rb file below supposed to be added back as part of this PR?